### DISCUSSION TITLE
Multiple modal/drawers

### DISCUSSION THREAD

**Discussion by dgastudio** [rsuite/rsuite#1585] [General]
Hello!

It's possible to open a new modal/drawer from already opened modal/drawer without replacing the existing one?

Thank you

**Comment by simonguo** [comment]
Modal and Drawer support overlay.

**Reply by dgastudio** [reply]
Hello, thank you for your response.
can you provide an example,. please?
because opening a new drawer from existing drawer, is replacing the current one instead of overlaying a new one

**Reply by dgastudio** [reply]
any advice?

**Reply by SevenOutman** [reply]
My practice is increasing the width of old drawer when opening a new drawer, so that it’s not completely covered.
